mvvmcDevSprint/Controllers/BarbaraBarone/BBLogin/BBLoginViewModel.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+
+final class BBLoginViewModel {
+    let endpoint = Endpoints.Auth.login
+    weak var viewController: BBLoginViewController?
+    
+    func fetchLogin(with email: String) {
+        let parameters = ["email" : email]
+        noInternetConnection()
+        viewController?.showLoading()
```

Comment:
essa linha 10 e todas pra baixo vai ser sempre executadas mesmo se n tiver internet
e deveria ser: só chama a API se tiver internet
tem que corrigir isso

Reply:
beleza!! eu fui ver esse meu erro quando tava mexendo na service, ja to corrigindo